client/src/pages/course/admin/mentors.tsx

```diff
@@ -22,11 +23,22 @@ function Page(props: CoursePageProps) {
 
   const service = useMemo(() => new CourseService(courseId), [courseId]);
 
+  const studentsValueName = ['Students with a mentor', 'Students who can have a mentor'];
+
   useAsync(async () => {
     setLoading(true);
     const records: any[] = await service.getMentorsWithDetails();
     const countries: Record<string, { totalCount: number }> = {};
 
+    const studentsGroupCount = records.reduce(
+      (acc, { studentsCount, maxStudentsLimit }) => {
+        acc[0] += studentsCount ? +studentsCount : 0;
```

Comment:
is there any reason for conversion (`+`)?

Reply:
Probably no reason